events/warlord_china_events.txt

```diff
@@ -559,16 +588,26 @@ warlord_china_events.100 = { #Split china up into warlord states
 			}
 			add_modifier = {
 			name = modifier_warlord_era
-			months = 300
+			months = -1
 			}
 		}
 		every_subject_or_below = {
 			make_independent = yes
 		}
-		ig:ig_armed_forces = {
-			set_interest_group_name = ig_zhili_clique
-		}
 
+		play_as = c:ZHI
```

Comment:
Just to be sure, you've tested you dont get a game over when you press this button?

Reply:
Yes, it does indeed tag switch correctly